src/core/internal/convert.d

```diff
@@ -10,13 +10,33 @@
 module core.internal.convert;
 import core.internal.traits : Unqual;
 
-@trusted pure nothrow
+/+
+A @nogc function can allocate memory during CTFE.
++/
+@nogc nothrow pure @trusted
+private ubyte[] ctfe_alloc()(size_t n)
```

Comment:
I suppose this is a template to avoid having it actually get stored in the object file?

Reply:
Yes, if it's unused. It doesn't prevent the function from appearing in the object file if it's used during CTFE, alas.